src/components/spacings/inline/inline.js

```diff
@@ -14,17 +14,29 @@ Inline.propTypes = {
   scale: PropTypes.oneOf(['xs', 's', 'm', 'l', 'xl']),
   alignItems: PropTypes.oneOf([
     'stretch',
+    'flex-start',
+    'flex-end',
+    'center',
+    'baseline',
+    // Deprecated
```

Comment:
Why is this in our diff here?

Reply:
I did a mistake when rebasing, I'll fix this.

Reply:
Rebased again and squashed the commits